web/src/scheduled_messages.js

```diff
@@ -52,12 +53,14 @@ export function add_scheduled_messages(scheduled_messages) {
     for (const scheduled_message of scheduled_messages) {
         scheduled_messages_data[scheduled_message.scheduled_message_id] = scheduled_message;
     }
+    scheduled_messages_ui.show_schedule_message_indicator();
```

Comment:
This data module isn't allowed to call into `scheduled_messages_ui` -- doing so would add another import cycle.

Simplest solution is to have the callers each call `scheduled_messages_ui`

Reply:
I ended up  extracting a new module since this version still had an import cycle.